src/disk/state.rs

```diff
@@ -399,6 +399,21 @@ impl Default for Xmrig {
     }
 }
 
+impl Default for Xvb {
+    fn default() -> Self {
+        Self {
+            simple: true,
+            token: "".to_string(),
+            simple_hero_mode: false,
+            mode: XvbMode::Auto,
+            manual_amount_raw: 0.0,
+            manual_slider_amount: 0.0,
+            manual_donation_level: ManualDonationLevel::Donor,
+            manual_donation_metric: ManualDonationMetric::Hash,
+        }
+    }
+}
+
```

Comment:
use ..default::Default() instead of writing every fields for the ones that still use the default value.

Reply:
did you forgot it ? you marked it as resolved.